README.md

```diff
@@ -6,20 +6,20 @@ Yesterday, a major collaboration between Microsoft, Chevron and Schlumberger, an
-), which the company boasts to be "the industry's largest in cloud computing." 
+), which the company says is "the industry's largest in cloud computing." 
 
-As Microsoft workers, whilst we are proud of the efforts the company has made towards improving sustainability we need to do more.
+As Microsoft workers, whilst we are proud of the efforts the company has made towards improving sustainability, we must do more.
 
 The Global Climate Strike is an opportunity for Microsoft to set an example as a leader in the tech industry. The company has taken many initiatives that focus on individual contributions, such as educating employees on composting, reducing waste, and making our [campus carbon neutral](https://blogs.microsoft.com/on-the-issues/2017/07/14/creating-carbon-free-headquarters-puget-sound-region/ ). There are also efforts to leverage artificial intelligence to solve specific and important climate-related problems. However, given the urgency and complexity of the crisis, these actions are not enough.
 
-If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. It's clear that the tech industry is one of the main culprits behind our burning planet, both in consuming a great deal of fossil fuels through power hungry datacenters and supporting Big Oil companies. But we also have to recognize that we're equipped with so many resources that we can leverage to steer our industry and government in the right direction. 
+If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. 
 
-Following our mission to "Empower every person and every organization in the planet to achieve more", it is essential that we secure a planet with livable conditions for *everyone*. This includes victims of [disaster displacement](https://weather.com/forecast-change/news/2019-06-04-how-the-middle-east-drought-and-flood-cycle-will-probably-lead-to ) often from the Global South and Black and Brown communities who are [disproportionately affected](https://www.ebony.com/news/environmental-justice-begins-with-understanding-how-climate-change-impacts-african-american-communities/) by climate change.
+Following our mission to "Empower every person and every organization in the planet to achieve more", it is essential that we secure a planet with livable conditions for *everyone*. 
```

Comment:
I think it's important to leave this part in

Reply:
Actually yes you're right that's a good sentence; doesn't blame anyone or divert from the main point the original poster had.

Reply:
Bumped, as I edited after initial reply. Hm I just realised I'm not as good at github as I should be. You mean, you'd want to retain the whole deleted part (line 22)?

Reply:
Yeah, I'd want to keep all of it, to be clear that
a) people are currently being affected by climate change, e.g. by disasters
b) the majority affected are the global south and black and brown communities

Reply:
I thought they were good points, too, but didn't think they were directly relevant to the post. I could imagine them being in an essay on the awful reality and social injustices of climate change, of course. Since neither of us can control the master, let's see what happens.

Reply:
I agree with @JPumphrey here that it's important to acknowledge the intersectional nature of climate change. If anything, I'd add even more to this section to also include the diversity and inclusivity core priority at Microsoft (which I feel these contracts violate).

Reply:
Your edits have a real "All Lives Matter" vibe that frankly seem in poor taste to me. I agree that climate change is currently being felt by marginalized communities to an apocalyptic degree, and the original text reflects that while your edits do not. Seems inappropriate.